routines/populate.go

```diff
@@ -0,0 +1,50 @@
+package routines
+
+import (
+	"log"
+	"time"
+
+	"github.com/csivitu/bl0b/ctftime"
+	"github.com/csivitu/bl0b/db"
+)
+
+func populate(t time.Time) {
+	log.Println("Populating database with upcoming CTFs!")
+	DB := db.New()
+	defer DB.Close()
+
+	ctf := ctftime.New()
+
+	now := t.Unix()
+	events, err := ctf.GetEvents(10, now, now+60*60*24*7)
```

Comment:
Let's just keep all function signatures (like `ctf.GetEvents` consistent and make them take `time.Duration`)

Also this is in seconds instead of millisecond? Could be simplified to `now+time.Hour*24*7`

Reply:
Yep, I can change the function signature for GetEvents to take a time object.

`t.Unix()` returns a timestamp in seconds, and CTFtime can interpret that correctly, so I kept it that way. Of course, Javascript would say that it's some date in 1970. Should I change it to milliseconds?

Reply:
I guess we should convert to milliseconds in order to avoid confusion. I'll do that and resolve this conversation.

Reply:
Refactored GetEvents, changed time to milliseconds. Resolving this :partying_face: